ext/fs/30_fs.js

```diff
@@ -267,14 +279,14 @@ function parseFileInfo(response) {
       ? new Date(response.birthtime)
       : null,
     dev: response.dev,
```

Comment:
```suggestion
    // Only non-null if on Unix
    dev: unix ? response.dev : null,
```

Maybe it would be good to revert the past PR then only include the code that should change. That will help restore it back to the way it was more accurately.